allan_variance.py

```diff
@@ -120,3 +129,23 @@ def params_from_avar(tau, avar):
     prediction = A.dot(x) * avar
 
     return np.sqrt(x), prediction
+
+
+def allan_deviation(avar):
```

Comment:
Ok. I think ``Reference`` in ``allan_variance`` is enough to know about  adev (https://github.com/dinya/allan-variance/blob/72ad710bd2f157d0eb9d92465f307b55cacc43d0/allan_variance.py#L61)